setup.py

```diff
@@ -12,6 +12,7 @@
     description='OpenID Connect Provider (OP) library in Python.',
     install_requires=[
         'oic >= 0.15.0',
-        'pymongo'
+        'pymongo',
+        'redis'
```

Comment:
I am thinking that we should put `pymongo` and `redis` as `extras_require`. Then users can choose the preferred backend/db.

Reply:
Do you want it to be two `try: import pymongo except ImportError: pass` blocks in storage.py?

Reply:
I think that is the simplest change. I'm fine with that.

Reply:
Please check the third commit in the patchset. Do you think that the documentation should be updated as well?

Reply:
I went ahead and merged this, but adding documentation would be great ;) of course it can be done in a separate PR. Thank you!